docs/resources/Channel.md

```diff
@@ -396,17 +396,17 @@ Represents a message sent in a channel within Discord.
 
 There are currently four situations in which a message has a message_reference object:
 
-Crosspost messages: messages that originated from another channel (IS_CROSSPOST flag).  
+Crosspost messages: messages that originated from another channel (IS_CROSSPOST flag).
 These messages have all three fields, with data of the original message that was crossposted.
 
-Channel Follow Add messages: automatic messages sent when a channel is followed into the current channel (type 12).  
+Channel Follow Add messages: automatic messages sent when a channel is followed into the current channel (type 12).
 These messages have the `channel_id` and `guild_id` fields, with data of the followed announcement channel.
 
-Pin messages: automatic messages sent when a message is pinned (type 6).  
+Pin messages: automatic messages sent when a message is pinned (type 6).
 These messages have `message_id` and `channel_id`, and `guild_id` if it is in a guild, with data of the message that was pinned.
 
-Replies: messages replying to a previous message (type 19).  
-These messages have `message_id` and `channel_id`, and `guild_id` if it is in a guild, with data of the message that was replied to. The channel_id and guild_id will be the same as the reply.  
+Replies: messages replying to a previous message (type 19).
+These messages have `message_id` and `channel_id`, and `guild_id` if it is in a guild, with data of the message that was replied to. The channel_id and guild_id will be the same as the reply.
```

Comment:
these spaces at the ends are intentional, as 2 spaces at the end of a line is a markdown newline

anyway this section was replaced in the threads PR so editing it will cause merge conflicts

Reply:
Didn't mean to do this on purpose, was made by my linter. Will fix